src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -54,7 +54,7 @@ const alertStyles = ({theme, type, system, alignTop}) => css`
   ${system && getSystemAlertColorStyles(theme.alert[type])};
 `;
 
-const AlertWrapper = styled('div', {shouldForwardProp: isPropValid})`
+const AlertWrapper: any = styled('div', {shouldForwardProp: isPropValid})`
```

Comment:
Why is this `any`?

Reply:
I think you can prob type the `alertStyles` stuff like this: https://github.com/getsentry/sentry/pull/14703/files#diff-3d15e1f6a6441ac3fdf4f164638bcaefR21